### ISSUE TITLE
What happened to Releases?

### ISSUE THREAD

**Issue by mcejp** [felixjones/gba-toolchain#47]
Back in May, I downloaded release 3.0.4. But now it looks like [all releases are gone](https://github.com/felixjones/gba-toolchain/releases) and there are no Git tags. Is this intentional?

**Comment by felixjones** [comment]
Yes intentional to move away from releases. A lot of people were downloading old versions

If you want, I can restore them as their own branches so you can download them

**Comment by mcejp** [comment]
I guess it's not a huge deal, but I was trying to find out the commit hash of the version that I am using and had to resort to the [Internet Archive copy of the page](https://web.archive.org/web/20220821045846/https://github.com/felixjones/gba-toolchain/releases).

In absence of any releases, how about a small note in the Readme indicating whether the main branch is considered stable?

In any case, thanks for your work on this project :)

**Comment by tfinnegan937** [comment]
I would like to recommend the use of commit tags to fix this, as I would also very much like to be able to peg my code to a stable release.

Commit tags are nice for preventing folks from accidentally downloading the wrong version because you have to manually peg the tagged commit to a branch in your local repository, and then switch to it and pull it.

@felixjones  This might meet both the requester's intent, and your desire to keep people from downloading older versions.
